doc/building.rst

```diff
@@ -241,7 +265,9 @@ follow `Windows Installer Build (Cross-Compile)`_ instead.
 
    The Nextcloud binary will appear in the ``bin`` directory.
 
-Windows Installer Build (Cross-Compile)
+.. _`Windows Installer Build (Cross-Compile)`:
+
+Windows Installer (i.e. Deployment) Build (Cross-Compile)
```

Comment:
Same here. I think it's better to first merge the build instructions and then later in a separate pr focus on the installer instructions.